flink-jepsen/README.md

```diff
@@ -27,18 +27,55 @@ To simplify development, we have prepared Dockerfiles and a [Docker Compose](htt
 so that you can run the tests locally in containers (see Section [Docker](#usage-docker)).
 
 ### Running Tests
-This project does not comprise of only a single test that can be run but rather a parameterizable
-test template. This allows the user to specify the cluster manager that Flink should be on, the
-location of the high availability storage directory, the jobs to be submitted, etc.
-The script under `docker/run-tests.sh` shows examples on how to specify and run tests.
-By default, the example tests run the `DataStreamAllroundTestProgram`, which is located under
-`flink-end-to-end-tests/flink-datastream-allround-test` of the Flink project root.
-Before running the tests, you have to build the job first, and copy the resulting jar
-(`DataStreamAllroundTestProgram.jar`) to the `./bin` directory of this project's root.
-Also included in the examples is a more complicated scenario with two jobs that share a Kafka
-topic. See the `run-tests.sh` script for details on how to enable and run this test.
+Tests are started from the control node. Therefore, the contents of this directory
```

Comment:
This `README` assumes that the user is somewhat familiar with Jepsen. I can add something here or to _"Setting up the Environment"_.

Reply:
The environment section would be a good place.

It is true that someone following the Jepsen documentation link will not need it, but I think it would be nice if one could read the README in full and later dive into the details of Jepsen.
Basically, without knowing a lot about Jepsen I could make good enough sense things, but this part right here tripped me up a bit.

Reply:
Added additional paragraph.